locale/en/tag_names.json

```diff
@@ -0,0 +1,39 @@
+{
+    "@metadata": {
+		"authors": [
+			"Susana Cárdenas Molinar",
+			"Jason Sherman",
+			"Sam Walton"
+		],
+		"locale": "en",
+		"message-documentation": "qqq"
+	},
+    "language_translation": "language",
+    "multidisciplinary_translation": "multidisciplinary",
+    "physical-science_translation": "physical-science",
+    "music_translation": "music",
+    "art_translation": "art",
+    "social_translation": "social",
+    "sciences_translation": "sciences",
```

Comment:
I think that these two tags exist by mistake, and that the one resource that has them should probably be using the `social-sciences` tag.

Reply:
Those message keys make more sense. Have removed both tags.